iot/api-client/manager/README.md

```diff
@@ -68,6 +72,7 @@ Create an ES device:
     java -cp target/cloudiot-manager-demo-1.0-jar-with-dependencies.jar \
```

Comment:
Wow that's long

Reply:
I agree, updated the manager samples to use `mvn:exec`, much cleaner.